language/stdlib/modules/LibraAccount.move

```diff
@@ -266,13 +267,10 @@ module LibraAccount {
         // Check that the `to_deposit` coin is non-zero
         let deposit_value = Libra::value(&to_deposit);
         Transaction::assert(deposit_value > 0, 7);
-
-        // TODO: on-chain config for travel rule limit instead of hardcoded value
-        // TODO: nail down details of limit (specified in LBR? is 1 LBR a milliLibra or microLibra?)
-        let travel_rule_limit = 1000;
+        let travel_rule_limit_microlibra = TravelRuleLimit::get_cur_limit();
         // travel rule only applies for payments over a threshold
-        let above_threshold =
-            Libra::approx_lbr_for_value<Token>(deposit_value) >= travel_rule_limit;
+        let approx_lbr_microlibra_value = Libra::approx_lbr_for_value<Token>(deposit_value) * 1000;
```

Comment:
If the value in `TravelRuleLimit` is already  in microlibra (1000000), doesn't that mean you shouldn't multiply by 1000 here?

Reply:
ah, is Libra::value already microLibra? / I guess everywhere else too?

Reply:
Yeah, `Libra<T>::value` is always micro`T`. `approx_lbr_for_value` is just for converting between `T` and `LBR` (cc @tzakian to be sure)

Reply:
Sam is correct. The `approx_lbr_for_value` is meant to be a "base units of currency to base units of LBR" (microlibra).

Reply:
it seems elsewhere (for example Libra<CoinType> { value } is treated as LBR not microLBR.
As the threshold was hardcoded to 1000 previously, it seems like some tests use that 1000 value when it should have been 1_000_000?